confutils.nim

```diff
@@ -70,6 +70,8 @@ const
   confutils_description_width {.intdefine.} = 80
   confutils_narrow_terminal_width {.intdefine.} = 36
 
+{.push gcsafe, raises: [].}
```

Comment:
we tend to put this on top (so that it's part of the file "header" along with copyright comment)